etc/jitsi-meet-desktop.profile

```diff
@@ -0,0 +1,30 @@
+# Firejail profile for jitsi-meet-desktop
+# Description: Jitsi Meet desktop application powered by Electron
+# This file is overwritten after every install/update
+# Persistent local customizations
+include jitsi-meet-desktop.local
+# Persistent global definitions
+include jitsi-meet-desktop.local
+
+noblacklist ${HOME}/.config/Jitsi Meet
+
+include disable-common.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
```

Comment:
What about:
 - disable-devel
 - disable-exec
 - disable-interpreters
 - disable-xdg

Reply:
It doesn't launch with disable-exec. I've added the others

Reply:
Try `disable-exec.inc` and `ignore noexec ${HOME}` (maybe also `ignore noexec /tmp`).

The `ignore` commands should be put between `include globals.local` and `noblacklist` (with blank lines).